location/src/main/java/com/yas/location/config/ServiceUrlConfig.java

```diff
@@ -0,0 +1,8 @@
+package com.yas.location.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "yas.location")
+public record ServiceUrlConfig(String media, String product) {
```

Comment:
Is location going to communicate with media or product service?

Reply:
Thanks, I have fixed it, please review again.